File: sdkwork-backend-sdk-go/types/generate_character_vo.go
package types

// 角色生成结果VO
type GenerateCharacterVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	RequestId string `json:"requestId"`
	TaskId string `json:"taskId"`
	TaskStatus string `json:"taskStatus"`
	CharacterImages []ImageMediaResource `json:"characterImages"`
	CharacterId string `json:"characterId"`
	CharacterName string `json:"characterName"`
	CharacterDescription string `json:"characterDescription"`
	Model string `json:"model"`
	Duration int `json:"duration"`
	ErrorMessage string `json:"errorMessage"`
}

File: sdkwork-backend-sdk-csharp/Models/GenerateCharacterVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class GenerateCharacterVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? RequestId { get; set; }
        public string? TaskId { get; set; }
        public string? TaskStatus { get; set; }
        public List<ImageMediaResource>? CharacterImages { get; set; }
        public string? CharacterId { get; set; }
        public string? CharacterName { get; set; }
        public string? CharacterDescription { get; set; }
        public string? Model { get; set; }
        public int? Duration { get; set; }
        public string? ErrorMessage { get; set; }
    }
}
